constraintlayout/core/src/main/java/androidx/constraintlayout/core/widgets/analyzer/Direct.java

```diff
@@ -54,12 +58,18 @@
     public static void solvingPass(ConstraintWidgetContainer layout, BasicMeasure.Measurer measurer) {
         ConstraintWidget.DimensionBehaviour horizontal = layout.getHorizontalDimensionBehaviour();
         ConstraintWidget.DimensionBehaviour vertical = layout.getVerticalDimensionBehaviour();
+        hcount = 0;
+        vcount = 0;
+        long time = System.nanoTime();
```

Comment:
It should be 
long time = 0;
if (DEBUG) {
   time = System.nanoTime():
} 
For performance reasons